test/support/environment-variables.ts

```diff
@@ -41,10 +47,14 @@ export const getEnvironmentVariablesForProcessEnv = (): NodeJS.ProcessEnv => ({
   ACBS_AUTHENTICATION_RETRY_DELAY_IN_MILLISECONDS: ENVIRONMENT_VARIABLES.ACBS_AUTHENTICATION_RETRY_DELAY_IN_MILLISECONDS.toString(),
   ACBS_TIMEOUT: ENVIRONMENT_VARIABLES.ACBS_TIMEOUT.toString(),
   ACBS_USE_RETURN_EXCEPTION_HEADER: ENVIRONMENT_VARIABLES.ACBS_USE_RETURN_EXCEPTION_HEADER.toString(),
+  APIM_MDM_MAX_REDIRECTS: ENVIRONMENT_VARIABLES.APIM_MDM_MAX_REDIRECTS.toString(),
+  APIM_MDM_TIMEOUT: ENVIRONMENT_VARIABLES.APIM_MDM_TIMEOUT.toString(),
 });
 
 export const TIME_EXCEEDING_ACBS_TIMEOUT = ENVIRONMENT_VARIABLES.ACBS_TIMEOUT + 500;
 
 export const TIME_EXCEEDING_ACBS_AUTHENTICATION_TIMEOUT = ENVIRONMENT_VARIABLES.ACBS_AUTHENTICATION_TIMEOUT + 500;
 
 export const TIME_EXCEEDING_ACBS_AUTHENTICATION_ID_TOKEN_CACHE_TTL = ENVIRONMENT_VARIABLES.ACBS_AUTHENTICATION_ID_TOKEN_CACHE_TTL_IN_MILLISECONDS + 100;
+
+export const TIME_EXCEEDING_MDM_TIMEOUT = ENVIRONMENT_VARIABLES.APIM_MDM_TIMEOUT + 500;
```

Comment:
Why do we add 500 to the timeout environment variables for testing?

Reply:
`TIME_EXCEEDING_MDM_TIMEOUT` is the number of milliseconds to delay responses from MDM when we want to test what happens if our requests to MDM time out, so it needs to be something larger that `ENVIRONMENT_VARIABLES.APIM_MDM_TIMEOUT`.

When we did this for ACBS, we picked 500 quite arbitrarily but it has worked fine.

Now that you've mentioned it, we could probably reduce it without any issue and it would speed up the API tests slightly. I'll do that